vuepress/docs/next/tutorials/create/ms/spring-boot-ms.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebarDepth: 2
+---
+
+# Create a Spring Boot Microservice
+
+This tutorial uses the Spring Initializr to create a simple microservice to quicky generate an Entando bundle project. 
+
+## Prerequisites
+- [A working instance of Entando](../../../docs/getting-started/)
+- Verify dependencies with the [Entando CLI](../../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Step 1: Initialize the Bundle with a Microservice
+
+1. Initialize a new bundle: 
+```
+ent bundle init e71-spring-project
+```
+2. From the bundle project root directory, create a new microservice:
+```
+cd e71-spring-project
+ent bundle ms add spring-ms
+```
+
+## Step 2: Create the Microservice
+1. At the Spring Initialyzr page, [start.spring.io](http://start.spring.io/), generate a project with the following configuration:
+```
+	Project=Maven
+	Language=Java
+	Spring Boot version=2.7.6
+	Group=com.entando.example
+	Artifact=spring-ms 
+	Name=spring-ms 
+	Description=Demo project for Spring Boot
+	Package name=com.entando.example.spring-ms
+	Packaging=Jar
+	Java=11
+	Dependencies= #under WEB: Spring Web 
+				  #under OPS: Spring Boot Actuator
+```
+
```

Comment:
maybe include the step to generate the zip? usually we mention clicking whatever button does something with a form